extended-tests/src/test/kotlin/org/stellar/anchor/platform/extendedtest/auth/apikey/custody/AuthApiKeyCustodyTests.kt

```diff
@@ -1,43 +1,18 @@
-package org.stellar.anchor.platform
+package org.stellar.anchor.platform.extendedtest.auth.apikey.custody
 
 import java.util.concurrent.TimeUnit
 import kotlin.test.assertEquals
 import okhttp3.OkHttpClient
 import okhttp3.Request
 import okhttp3.RequestBody
-import org.junit.jupiter.api.AfterAll
-import org.junit.jupiter.api.BeforeAll
 import org.junit.jupiter.params.ParameterizedTest
 import org.junit.jupiter.params.provider.CsvSource
+import org.stellar.anchor.platform.extendedtest.auth.AbstractAuthIntegrationTest
 import org.stellar.anchor.util.GsonUtils
 import org.stellar.anchor.util.OkHttpUtil
 
-internal class CustodyApiKeyAuthIntegrationTest : AbstractAuthIntegrationTest() {
-  companion object {
-    @BeforeAll
-    @JvmStatic
-    fun setup() {
-      println("Running CustodyApiKeyAuthIntegrationTest")
-      testProfileRunner =
-        TestProfileExecutor(
-          TestConfig(testProfileName = "default-custody").also {
-            it.env[RUN_DOCKER] = "true"
-            it.env[RUN_ALL_SERVERS] = "false"
-            it.env[RUN_CUSTODY_SERVER] = "true"
-
-            // enable custody server api_key auth
-            it.env["custody_server.auth.type"] = "api_key"
-          }
-        )
-      testProfileRunner.start()
-    }
-
-    @AfterAll
-    @JvmStatic
-    fun breakdown() {
-      testProfileRunner.shutdown()
-    }
-  }
+// use TEST_PROFILE_NAME = "auth-apikey-custody"
```

Comment:
This is an idea: is there a way to make the tests fail if they run against an unexpected profile?

Reply:
The test will naturally fail if it is run against an unexpected profile. The old paradigm of running the docker compose with the test will be able to make sure of the consistency. However, we have switched to the new paradigm.

Another alternative is to have an end-point to expose the test-profile. However, this may cause other issues. WDYT?

Reply:
Yeah, on second thought, I don't think we should be doing this.